_includes/tutorials/kafka-producer-application/kafka/markup/dev/create-topic.adoc

```diff
@@ -0,0 +1,17 @@
+
+In this step we're going to create a topic for use during this tutorial.
```

Comment:
It reads strangely to say we're going to do something but then do something else.  So FYC, remove this line.  Then see comments below